services/token_update.proto

```diff
@@ -135,4 +135,10 @@ message TokenUpdateTransactionBody {
      * currently have this key, transaction will resolve to TOKEN_HAS_NO_FEE_SCHEDULE_KEY
      */
     Key fee_schedule_key = 14;
+
+    /**
+     * The Key which can pause and unpause the Token. If the Token doesnt not currently have a pause key,
```

Comment:
Does the pause key need to be used to sign the update transaction?

Reply:
just the Admin Key is enough I think